python/pecos/reps/pypmir/pypmir.py

```diff
@@ -229,10 +245,14 @@ def from_phir(cls, phir: dict, name_resolver=None) -> PyPMIR:
                 name = o["data"]
 
                 if name == "cvar_define":
+                    data_type = o["data_type"]
+                    size = int(data_type[1:])
+                    if "size" in o:
+                        size = o["size"]
```

Comment:
This takes care of default sizes for signed integers, but overrides them if specified.